src/store/entities/Member.java

```diff
@@ -2,4 +2,122 @@
 
 public class Member {
 
+	private String name;
+	private String address;
+	private String phoneNumber;
+	private String dateJoined;
+	private double feePaid;
+	private int id;
```

Comment:
I think you want the id to be a String, to allow for alphanumerical representation.